js/adapt-offlineStorage-xapi.js

```diff
@@ -58,11 +58,12 @@ define([
     },
 
     /**
-     * @returns {{name: string in the format Firstname Lastname, firstname: string, lastname: string }}
+     * @returns {{id: string, name: string in the format Firstname Lastname, firstname: string, lastname: string }}
      */
     getLearnerInfo: function() {
       var actor = xapi.get('actor') || {};
       var name = actor.name || '';
+      var id = actor.account && actor.account.name;
```

Comment:
Maybe the `id` should be something like `mbox`, `openid` or we could add a new config setting to the extension to let the user specify?